CHANGELOG.md

```diff
@@ -0,0 +1,39 @@
+Changelog
+=========
+
+## 1.8.3~ynh7 ([2019-03-12](https://github.com/YunoHost-Apps/leed_ynh/pull/18))
+
+- [Normalization from example_ynh](https://github.com/YunoHost-Apps/leed_ynh/commit/1418b13ff9c3d2f1849ad21b5c645609e3bea863)
+
+
+## 1.8.3~ynh6 ([2019-02-03](https://github.com/YunoHost-Apps/leed_ynh/pull/17))
+
+- [Add progression bar](https://github.com/YunoHost-Apps/leed_ynh/commit/9085643892836479b8c33bba6f5054512e4d441f)
+* [Changelog & html email](https://github.com/YunoHost-Apps/leed_ynh/commit/298f9ebf5767be9c6e163708cc69121a67aca1ed)
```

Comment:
Why * here and - above?

Reply:
I don't remember if I've explained it anywhere. The * is used instead of - to indicate a significative change. Something users would really like to know, like a version upgrade or other important changes.
It won't change the way the log is shown, but those lines will be include into the light version of the log.
Have a look to the parser for more information about that.
If I haven't yet, an explanation should be added to this helper